README.md

```diff
@@ -1,42 +1,80 @@
+
+This repo serves as a discussion board for additions and improvements to Immersive Web standards.
+
+While much of our work to date has been on the WebXR Device API, this repo also holds proposals for features that could end up in other standards.
+
+If your proposal isn't in some way related to Immersive Web technologies like Virtual/Augmented/Mixed Reality then head over to the [W3C Community and Business Groups](https://www.w3.org/community/) to find an appropriate group. The [Web Incubator Community Group (WICG)](https://wicg.io/) is also a good general-purpose venue.
+
+## Code of Conduct
+
+**TL;DR: Treat other participants with respect or get banned.**
+
+This repository is for *open discussion of ideas* for making the Immersive Web better and we foster by an environment where all discussion is respectful. Every person participating in the community group is expected to follow the [Code of Conduct](https://github.com/immersive-web/community-resources/blob/master/CODE_OF_CONDUCT.md).
+
+## How does a feature idea become a standard?
+
+An immersive web feature goes through several steps to move from idea to standard:
+
+- File a Proposals Issue
+- Find Consensus for the Idea
+- Recruit Feature Advocates
+- Create a Feature Repo
+- Write an Explainer
+- Find Consensus for Feature Details
+- Recruit Implementers
+- Pitch a Working Group
+
+**Most of the work of landing a feature is social, not technical.**
+
+If you're reading this, you probably have ideas about what "should" be in the Immersive Web. Welcome, there are a lot of dreamers in our community!
+
+That said, landing a feature in a standard consists mostly of writing documents, having conversations, and eventually finding consensus for an idea. You'll need to rally support from people who can make prototypes, write documentation, and then as a group you'll work to convince a working group that the feature is important enough to take through the W3C standard.
 
 ## I have a proposal! How do I get started?
-To get the ball rolling with a proposal, file an issue on this repo. That issue will host discussion for the proposal until such time that it's either given its own repo or closed as will not implement. Note that the issue does not have to have sustained discussion to remain open: It may take a while for some proposals to attract the attention necessary to move forward, and that's OK!
 
-## Do I have to have a spec/explainer/proof of concept before posting an proposal?
-No! Certainly if you have any of those available it's great to share them, but it's absolutely not necessary. At minimum you should have a clear goal in mind and a proposed direction for how to address it.
+Great! If the paragraphs above haven't scared you off and you're up for the social work of standardization then you should file a GitHub Issue in the [proposals repo](https://github.com/immersive-web/proposals/issues) using the [Proposals Issue Template](ProposalsIssueTemplate.md).
```

Comment:
I would re-pitch that they should do a search first.  Or at least say "if the above hasn't scared you off, and you've already done a search..."